### ISSUE TITLE
pip install instructions for TACL and tsd-s3cmd

### ISSUE THREAD

**Issue by ofrei** [unioslo/tsd-api-client#130]
I've noticed difference in current install & update instructions  for TACL and tsd-s3cmd:

https://github.com/unioslo/tsd-api-client/blob/master/README.md
```
pip3 install tsd-api-client
pip3 install tsd-api-client --upgrade # to get the latest version
```

https://github.com/unioslo/tsd-s3cmd/blob/master/README.md
```
pip3 install <source code URL you copied above>
```

Few questions about this inconsistency:

* is this correct that installation should be done differently? e.g. is tsd-s3cmd package updated on pypi?
* could you update tsd-s3cmd README with instructions on how to update? (it only lists steps for install, while TACL lists both install and update
* it would be helpful to include ``tacl --version`` command in the README file next to install / update instructions so that users know how to check if they should update to a newer version 
* I'm using ``pip install <package>`` rather than ``pip3 install <package>``, because in my environment pip3 refers to system-wide python, while pip refers to my miniconda3 installation (see below); consider changing instructions to use ``pip install`` - and then users can change to ``pip3`` if this is relevant to their envitonment. 

```
(base) [oleksandr@ip113 run04]$ which pip
~/miniconda3/bin/pip
(base) [oleksandr@ip113 run04]$ which pip3
/usr/bin/pip3
(base) [oleksandr@ip113 run04]$ which python
~/miniconda3/bin/python
(base) [oleksandr@ip113 run04]$ which python3 
~/miniconda3/bin/python3
(base) [oleksandr@ip113 run04]$ python3 --version
Python 3.7.4
(base) [oleksandr@ip113 run04]$ python --version
Python 3.7.4
```

**Comment by haatveit** [comment]
>is this correct that installation should be done differently? e.g. is tsd-s3cmd package updated on pypi?

Yes, this is correct. We don't have plans for publishing tsd-s3cmd to PyPI.

 
>could you update tsd-s3cmd README with instructions on how to update? (it only lists steps for install, while TACL lists both install and update

If there are any further updates to tsd-s3cmd, we can add a note upgrades at the same time.

>it would be helpful to include `tacl --version` command in the README file next to install / update instructions so that users know how to check if they should update to a newer version

Good idea.

>I'm using `pip install <package>` rather than `pip3 install <package>`, because in my environment pip3 refers to system-wide python, while pip refers to my miniconda3 installation (see below); consider changing instructions to use `pip install` - and then users can change to `pip3` if this is relevant to their envitonment.

This is more of a failure of miniconda3 than our instructions for not registering a pip3 entrypoint IMO.

I've come to prefer the `python -m pip` or `python3 -m pip` style of invocation over `pip`/`pip3`, since you then have better control of which Python interpreter's pip entrypoint you are calling.

If I were redoing the installation section I'd prefer to point casual users towards [pipx](https://github.com/pypa/pipx), as it automates the setup of application-specific virtual environments.
